### PR TITLE
add test for wget

### PR DESCRIPTION
**Pull request by stephengroat**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wget.rb [hunk 1]
```diff
@@ -33,6 +34,8 @@ class Wget < Formula
   depends_on "libmetalink" => :optional
   depends_on "gpgme" => :optional
 
+  patch :p0, :DATA
```
Comment (fxcoudert) [inline Formula/wget.rb]: If the test fails, we need it reported upstream and use the upstream commit as a `patch`

Comment (stephengroat) [inline Formula/wget.rb]: will do, working out the patch

Comment (stephengroat) [inline Formula/wget.rb]: added

**Comment by MikeMcQuaid** [comment]
@stephengroat I think we talked about this before, no? We don't want `make check` tests as they are inferior to `test do` ones and don't actually catch bugs. They are for upstream to be running as part of CI and the release process and not us. The tiny exceptions may be crypto software. That this requires a patch to work should indicate that this test is not useful.

**Pull request closed without merging**